patches/tModLoader/Terraria/release_extras/tMLMod.targets

```diff
@@ -4,6 +4,7 @@
 		<TerrariaSteamPath>$(MSBuildThisFileDirectory)</TerrariaSteamPath>
 		<tMLSteamPath>$(MSBuildThisFileDirectory)</tMLSteamPath>
 		<tMLLibraryPath>$(tMLSteamPath)Libraries</tMLLibraryPath>
+		<tMLAnalyzerPath>$(tMLSteamPath)Analyzers</tMLAnalyzerPath>
 		<tMLName>tModLoader</tMLName>
```

Comment:
No need for a new folder, output analyzers into `Libraries`, the way `tModPorter` and `tModLoader.BuildTasks` get put there:
![image](https://github.com/tModLoader/tModLoader/assets/6795251/7d224d6d-d18a-47b3-8cfd-663f556e64c4)